pipelines/controller/src/main/java/com/google/fhir/analytics/HiveTableManager.java

```diff
@@ -52,16 +59,14 @@ public HiveTableManager(String jdbcUrl, String user, String password) {
    *     the THRIFT_CONTAINER_PARQUET_DIR directory.
    * @throws SQLException
    */
-  public void createResourceAndCanonicalTables(
+  public synchronized void createResourceAndCanonicalTables(
       List<String> resources, String timestamp, String thriftServerParquetPath)
       throws SQLException {
     if (resources == null || resources.isEmpty()) {
       return;
     }
 
-    // TODO: Make use of JdbcConnectionUtil to create jdbc connection
-    //  (https://github.com/google/fhir-data-pipes/issues/483)
```

Comment:
Query : I am assuming both the issues mentioned in this [issue](https://github.com/google/fhir-data-pipes/issues/483) will be addressed now, because we use pooled connections now as there is no need to create and close connection every time.

Reply:
Unfortunately not, but #483 is irrelevant to the above TODO (which is being addressed in this PR). We should still track/fix #483.